app/grandchallenge/evaluation/views/__init__.py

```diff
@@ -607,6 +607,38 @@ class EvaluationStatusBadgeDetail(ObjectPermissionRequiredMixin, DetailView):
     raise_exception = True
 
 
+class EvaluationIncompleteJobsDetail(
+    ObjectPermissionRequiredMixin, DetailView
+):
+    permission_required = "view_evaluation"
```

Comment:
In the template this data was only available to people with the `change_challenge` perm, so we should keep that. As we discussed, using cross object permissions is a bad idea. Fortunately, we assign the `change_evaluation` permission to the challenge admins group so can use that instead.